<launch>

  <node pkg="map_server" type="map_server" name="map_server" args="$(find shared_control)/map/map.yaml"/>
  <node pkg="shared_control" type="spatial_info_manager.py" name="spatial_info_manager" output="screen">
    <param name="gvd_PM" value="10.0"/>
    <param name="gvd_BM" value="3.74"/>
    <param name="gvg_minimum_path_distance" value="0.1"/>
    <param name="marker_cycle" value="2.0"/>
  </node>

  <!-- <node pkg="shared_control" type="fake_robot.py" name="fake_robot" output="screen">
    <param name="sim_cycle" value="0.1"/>
    <param name="robot_x" value="0.8"/>
    <param name="robot_y" value="-1.2"/>
    <param name="robot_th" value="87"/>
    <param name="robot_velocity" value="0.5"/>
    <param name="goal_margin" value="0.01"/>
  </node> -->
  <node pkg="shared_control" type="fake_bci.py" name="fake_bci" output="screen"/>

  <node pkg="shared_control" type="task_planner.py" name="task_planner" output="screen">
    <param name="spin_cycle" value="0.1"/>
    <param name="planning_cycle" value="0.3"/>
    <param name="goal_margin" value="0.2"/>
  </node>

</launch>
